src/audio/tdfb/tdfb.c

```diff
@@ -367,11 +667,21 @@ static int tdfb_cmd(struct comp_dev *dev, int cmd, void *data,
 
 	switch (cmd) {
 	case COMP_CMD_SET_DATA:
+		comp_dbg(dev, "tdfb_cmd(): COMP_CMD_SET_DATA");
 		ret = tdfb_cmd_set_data(dev, cdata);
```

Comment:
ditto in this function, even though it isn't something really for this patch...

Reply:
done